Mathlib/GroupTheory/FreeGroup/Reduce.lean

```diff
@@ -236,6 +247,19 @@ theorem toWord_inv (x : FreeGroup α) : x⁻¹.toWord = invRev x.toWord := by
   rcases x with ⟨L⟩
   rw [quot_mk_eq_mk, inv_mk, toWord_mk, toWord_mk, reduce_invRev]
 
+@[to_additive]
+theorem reduce_append : (reduce (L₁ ++ L₂)) = reduce (reduce L₁ ++ reduce L₂) := by
```

Comment:
This isn't a simp lemma, but I still think this equality and the one below should be reversed.